src/index.js

```diff
@@ -494,6 +504,7 @@ function createConfig(options, entry, format, writeMeta) {
 									emitDeclarationOnly: options.generateTypes && !useTypescript,
 									declarationDir: getDeclarationDir({ options, pkg }),
 									jsx: 'preserve',
+									esModuleInterop: !!tsconfigOptions.esModuleInterop,
```

Comment:
I don't really understand the reason for this? You're setting a default to come from the user's TSConfig, but the user's TSConfig overrides these defaults.

Reply:
good catch, this is kinda duplicated, I've removed it